src/window.rs

```diff
@@ -590,41 +590,52 @@ pub trait Window: AsAny + Send + Sync {
     // extension trait
     fn reset_dead_keys(&self);
 
-    /// Returns the position of the top-left hand corner of the window's client area relative to the
-    /// top-left hand corner of the desktop.
+    /// The position of the top-left hand corner of the surface relative to the top-left hand corner
+    /// of the window.
     ///
-    /// The same conditions that apply to [`Window::outer_position`] apply to this method.
+    /// This, combined with [`outer_position`], can be useful for calculating the position of the
+    /// surface relative to the desktop.
     ///
-    /// ## Platform-specific
+    /// This may also be useful for figuring out the size of the window's decorations (such as
+    /// buttons, title, etc.), but may also not correspond to that (e.g. if the title bar is made
+    /// transparent using [`with_titlebar_transparent`] on macOS, or your are drawing window
+    /// decorations yourself).
     ///
-    /// - **iOS:** Returns the top left coordinates of the window's [safe area] in the screen space
-    ///   coordinate system.
-    /// - **Web:** Returns the top-left coordinates relative to the viewport. _Note: this returns
-    ///   the same value as [`Window::outer_position`]._
-    /// - **Android / Wayland:** Always returns [`RequestError::NotSupported`].
+    /// This _may_ be negative in the off case that the windowing system decided to make a surface
+    /// that is larger than your window.
```

Comment:
I don't think it's how it's generally, the offset could be negative because it just decided to do so, it doesn't mean that it's bigger and the whole sentence could be dropped.

Or you can rephase it to say that it could be negative (and that's about it), but that's already clear from the `i32` type we're using.

Reply:
I just kinda wanted to make it explicit, but I'll try to reword.